### PR TITLE
Code Coverage

### PR DESCRIPTION
**Pull request by srowhani**
#patch#

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
     "ember-browserify": "1.1.11",
     "ember-cli": "~2.5.0",
     "ember-cli-app-version": "^1.0.0",
-    "ember-cli-code-coverage": "0.3.2",
+    "ember-cli-code-coverage": "srowhani/ember-cli-code-coverage",
```
Comment (sglanzer) [inline package.json]: Why the fork?

Comment (srowhani) [inline package.json]: The excludes prop wasn't working correctly. Tried some stuff in a fork, and got it to work. On my machine, the coverage works fine, but coveralls can't seem to parse the `lvoc.info` that my fork produces.

**Comment by sglanzer** [comment]
Can you PR the excludes fix?  I need that for my cases as well :)

**Comment by srowhani** [comment]
It still seems broken, I'd want to play around with it a bit more and get it working before I open a PR

**Pull request closed without merging**